src/test/rustdoc/intra-link-pub-use.rs

```diff
@@ -0,0 +1,17 @@
+#![deny(broken_intra_doc_links)]
+
+/// [std::env] [g]
+// FIXME: This can't be tested because rustdoc doesn't show documentation on pub re-exports.
```

Comment:
As of #63048 it does but only when inlining from an external crate.

Reply:
Nice! I'll add a test using that :)